src/test/java/EditingNotesScenarios.java

```diff
@@ -0,0 +1,71 @@
+import business_objects.Note;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import utils.androiddriver.AndroidDriverSingletone;
+import utils.dataProviders.NoteDataProvider;
+
+public class EditingNotesScenarios extends BaseTest{
+
+    @Test(dataProviderClass = NoteDataProvider.class, dataProvider = "EditNoteTitle")
+    public void changeNoteTitle(Note editedNote, Note desiredNote) {
```

Comment:
Again. It's a common practice to add any 'assertion' verb in test name: 
check
verify
etc..